Samples/WinMLSamplesGallery/TransformAsync/SegmentModel.cpp

```diff
@@ -0,0 +1,279 @@
+#include "pch.h"
+#include "SegmentModel.h"
+#include <iostream>
+#include <filesystem>
+
+using winrt::Windows::Foundation::PropertyValue;
+using winrt::hstring;
+using namespace winrt;
+using namespace Windows::Foundation::Collections;
+
+enum OnnxDataType : long {
+	ONNX_UNDEFINED = 0,
+	// Basic types.
+	ONNX_FLOAT = 1,
+	ONNX_UINT8 = 2,
+	ONNX_INT8 = 3,
+	ONNX_UINT16 = 4,
+	ONNX_INT16 = 5,
+	ONNX_INT32 = 6,
+	ONNX_INT64 = 7,
+	ONNX_STRING = 8,
+	ONNX_BOOL = 9,
+
+	// IEEE754 half-precision floating-point format (16 bits wide).
+	// This format has 1 sign bit, 5 exponent bits, and 10 mantissa bits.
+	ONNX_FLOAT16 = 10,
+
+	ONNX_DOUBLE = 11,
+	ONNX_UINT32 = 12,
+	ONNX_UINT64 = 13,
+	ONNX_COMPLEX64 = 14,     // complex with float32 real and imaginary components
+	ONNX_COMPLEX128 = 15,    // complex with float64 real and imaginary components
+
+	// Non-IEEE floating-point format based on IEEE754 single-precision
+	// floating-point number truncated to 16 bits.
+	// This format has 1 sign bit, 8 exponent bits, and 7 mantissa bits.
+	ONNX_BFLOAT16 = 16,
+}OnnxDataType;
+
+
+const int32_t opset = 12;
+
+/****	Style transfer model	****/
+void StyleTransfer::InitializeSession(int w, int h)
```

Comment:
>int w, int h [](http://example.com/codeflow?start=38&length=12)

🤔 Unused?

Reply:
It's inherited from streammodelbase, just to match the correct function signature.

Reply:
But is not surprising to the caller to pass 640x480, for `InitializeSession` to just ignore the values and use 720x720 anyway?

Should we: `SetImageSize(h, w);`?